### ISSUE TITLE
layout of missing emoji character raises an exception

### ISSUE THREAD

**Issue by JINZO631** [sassembla/LayouTaro#6]
# Reproduction steps

If you rewrite the sample as follows, an exception will be raised
`\U0001F971` is missing character.

```LayouTaroSample.cs
        var box = BoxElement.GO(
            null,// UI bg with image
            () =>
            {
                Debug.Log("root box element is tapped.");
            },
            TextElement.GO("\U0001F971\U0001F60A"),// 🥱😊
            ImageElement.GO(null),// image.
            ButtonElement.GO(null, () => { Debug.Log("button is tapped."); })
        );
```

# Expected results.

![image](https://user-images.githubusercontent.com/9525135/78856257-35211b80-7a61-11ea-8c6a-f84baf320958.png)

**Comment by sassembla** [comment]
Thank you for reporting! I'll fix it.

**Comment by sassembla** [comment]
Done! please wait next release with this fix.

![fixed](https://user-images.githubusercontent.com/944441/78859765-4e2eca00-7a6b-11ea-928f-54662b9db460.png)

**Comment by JINZO631** [comment]
layout of non-surrogate emoji character raises an StackOverflowException

# Reproduction steps

1. Rewrite EmojiOne.asset Unicode to 0x26A1.
![image](https://user-images.githubusercontent.com/9525135/78860847-2c831200-7a6e-11ea-93f6-2e9c3c4b66d5.png)

2. Layout the text containing "\u26A1"(⚡).

```LayouTaroSample.cs
        // generate your own data structure with parameters for UI.
        var box = BoxElement.GO(
            null,// UI bg with image
            () =>
            {
                Debug.Log("root box element is tapped.");
            },
            TextElement.GO("\u26A1"),// ⚡.
            ImageElement.GO(null),// image.
            ButtonElement.GO(null, () => { Debug.Log("button is tapped."); })
        );
```

**Comment by sassembla** [comment]
thank you for information! fixed in d2f4fe405e3ea0801dae420395f2cddb3806a122.
